script/Arcueid/Day 11/11_00_ARC11_1B.txt

```diff
@@ -72,7 +72,7 @@ I should let just it all go.
 [sha:803c0c8be2764a5f51450476350654c139d6f6f7]{
 -- Page 34, Offset 29579.
 -- ここで目を閉じるのが、まっとうな人間らしい選択だ。
-Pretending none of it ever happened would be the proper, human, choice to make.
+Pretending none of it ever happened would be the proper, human choice to make.
```

Comment:
Kinda like the comma here to emphasize human

Reply:
Maybe put italics instead? :/